pkg/xray/resource_xray_license_policy.go

```diff
@@ -68,14 +69,16 @@ func resourceXrayLicensePolicyV2() *schema.Resource {
 										Type:     schema.TypeList,
```

Comment:
You need descriptions on all of these fields - especially so you can generate docs

Reply:
done